tests/MailChimp/NewsletterTest.php

```diff
@@ -264,4 +264,25 @@ public function is_can_create_a_campaign()
 
         $this->newsletter->createCampaign($fromName, $replyTo, $subject, $html, $listName, $options, $contentOptions);
     }
+
+    /** @test */
+    public function it_will_throw_an_exception_when_using_a_subscribe_status_that_is_invalid()
```

Comment:
I believe that's already addressed in the test `it_can_override_the_defaults_when_subscribing_someone` on line 111, yes?
https://github.com/dodecasphere/laravel-newsletter/blob/default-status/tests/MailChimp/NewsletterTest.php#L111